drivers/dma/dma_stm32.h

```diff
@@ -44,17 +44,17 @@ struct dma_stm32_config {
 	struct dma_stm32_stream *streams;
 };
 
-#ifdef CONFIG_DMA_STM32_V1
```

Comment:
With stm32H7, DMA V1 has a DMAMUX (not only DMA V2). However for soc with DMA V2 and NO DMAMUX, the _stream id is in range 1..<dma-requests>_

Reply:
Is that resolved ?

Reply:
Yes. Now, the stream offset is 1, if we use dmamux or dma_v2. In the past, the offset was 0 if we dma_v1 which is not true for h7 socs. By the way, in the future it needs to be refactored in order to add `bdma` support which is dma_v2 controller. So a way to use dma_v1 in parallel to dma_v2 should be possible